stac_model/output.py

```diff
@@ -6,30 +6,11 @@
 from stac_model.base import DataType, MLMBaseModel, ModelTask, OmitIfNone, ProcessingExpression
 
 
-class ModelResult(MLMBaseModel):
+class ResultStructure(MLMBaseModel):
     shape: List[Union[int, float]] = Field(..., min_items=1)
     dim_order: List[str] = Field(..., min_items=1)
     data_type: DataType
```

Comment:
Can you add a `ModelResult = ResultStructure` alias just after?
This will avoid breaking existing code that could import it directly.